app/jobs/scraper/washington_job.rb

```diff
@@ -0,0 +1,55 @@
+class Scraper::WashingtonJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    url_base = "https://secure.lni.wa.gov/arts-public/"
+    browser = Watir::Browser.new :chrome,
+      options: {args: %w[--headless --no-sandbox --disable-dev-shm-usage --disable-gpu]}
+    browser.goto(url_base + "#/program-search")
+    js_doc = browser.element(css: "div.lni-u-flex.lni-u-flex-wrap.lni-u-items-end").wait_until(&:present?)
+    js_doc.button(text: "Search").click
+
+    next_button = browser.a(aria_label: "Next").wait_until(&:present?)
+    program_ids = []
+
+    loop do
+      find_table = browser.element(css: "tbody").wait_until(&:present?)
+
+      begin
+        table = Nokogiri::HTML(find_table.inner_html)
+      rescue Watir::Wait::TimeoutError
+        sleep 60
+        retry
```

Comment:
@jenlu33 is this timeout happening frequently? I wonder if we should only retry for X number of times so the job potentially doesn't get stuck forever. Something like this:

```
      begin
        retries ||= 0
        table = Nokogiri::HTML(find_table.inner_html)
      rescue Watir::Wait::TimeoutError
        sleep 60
        if (retries += 1) <= MAX_RETRIES
          retry
        else
          # Raise error or do nothing 
        end
      end
```

Reply:
It doesn't happen very often, I'll add that in